<launch>
    <!-- Main Formation Controller -->
    <node name="pde_formation_controller" pkg="pde_formation_control" type="formation_controller" output="screen">
        <!-- Circle trajectory parameters -->
        <param name="circle_radius" value="0.8"/>
        <param name="angular_frequency" value="0.3"/>
        <param name="center_x" value="0.0"/>
        <param name="center_y" value="0.0"/>
    </node>
</launch>